src/commands/scanner/run/dfa.ts

```diff
@@ -37,118 +29,56 @@ export default class Dfa extends ScannerRunCommand {
 		...ScannerRunCommand.flags,
 		// BEGIN: Filter-related flags.
 		'with-pilot': Flags.boolean({
-			summary: messages.getMessage('flags.withpilotSummary'),
-			description: messages.getMessage('flags.withpilotDescription')
+			summary: getMessage(BundleName.RunDfa, 'flags.withpilotSummary'),
+			description: getMessage(BundleName.RunDfa, 'flags.withpilotDescription')
 		}),
 		// END: Filter-related flags.
 		// BEGIN: Flags for targeting files.
 		// NOTE: All run commands have a `--target` flag, but they have differing functionalities,
 		// and therefore different descriptions, so each command defines this flag separately.
 		target: Flags.custom<string[]>({
 			char: 't',
-			summary: messages.getMessage('flags.targetSummary'),
-			description: messages.getMessage('flags.targetDescription'),
+			summary: getMessage(BundleName.RunDfa, 'flags.targetSummary'),
+			description: getMessage(BundleName.RunDfa, 'flags.targetDescription'),
 			required: true,
 			delimiter: ',',
 			multiple: true
 		})(),
 		// END: Flags for targeting files.
 		// BEGIN: Config-overrideable engine flags.
 		'rule-thread-count': Flags.integer({
-			summary: messages.getMessage('flags.rulethreadcountSummary'),
-			description: messages.getMessage('flags.rulethreadcountDescription'),
+			summary: getMessage(BundleName.RunDfa, 'flags.rulethreadcountSummary'),
+			description: getMessage(BundleName.RunDfa, 'flags.rulethreadcountDescription'),
 			env: 'SFGE_RULE_THREAD_COUNT'
 		}),
 		'rule-thread-timeout': Flags.integer({
-			summary: messages.getMessage('flags.rulethreadtimeoutSummary'),
-			description: messages.getMessage('flags.rulethreadtimeoutDescription'),
+			summary: getMessage(BundleName.RunDfa, 'flags.rulethreadtimeoutSummary'),
+			description: getMessage(BundleName.RunDfa, 'flags.rulethreadtimeoutDescription'),
 			env: 'SFGE_RULE_THREAD_TIMEOUT'
 		}),
 		// NOTE: This flag can't use the `env` property to inherit a value automatically, because OCLIF boolean flags
 		// don't support that. Instead, we check the env-var manually in a subsequent method.
-		[RULE_DISABLE_WARNING_VIOLATION_FLAG]: Flags.boolean({
-			summary: messages.getMessage('flags.ruledisablewarningviolationSummary'),
-			description: messages.getMessage('flags.ruledisablewarningviolationDescription')
+		'rule-disable-warning-violation': Flags.boolean({
```

Comment:
I have mixed feelings about this moving from a constant to a hardcoded string, but I have no alternative in mind aside from just importing the constant from the file it got moved to, which I also don't love. So this is the best option I guess.

Reply:
Yeah I was on the fence with this one as well. Since things moved around a bit - I think what we have here is fine... and is more readable.